src/services/FeedProcessor.ts

```diff
@@ -123,9 +129,25 @@ class FeedProcessor {
       return;
     }
 
+    let feedBase = '';
+    let feedQuote = '';
+    if (feedFetcher.base && feedFetcher.quote) {
+      feedBase = feedFetcher.base;
+      feedQuote = feedFetcher.quote;
+    } else {
+      const result = this.feedSymbolChecker.apply(feedFetcher.symbol);
+      if (result) {
+        [feedBase, feedQuote] = result;
+      } else {
+        this.logger.warn(`Cannot parse base & quote from symbol:${feedFetcher.symbol}`);
+        return;
+      }
+    }
+
     try {
       this.logger.debug(`${this.logPrefix} using "${feedFetcher.name}"`);
-      return (await fetcher.apply(feedFetcher.params, timestamp)) || undefined;
+      const params = Object.assign({...((feedFetcher.params ?? {}) as object)}, {feedBase, feedQuote});
+      return (await fetcher.apply(params, timestamp)) || undefined;
```

Comment:
```suggestion
      return (await fetcher.apply(params, timestamp));
```

idk why i was `undefined`, make no sense

Reply:
resolved